### DISCUSSION TITLE
引用河南一则业主的停贷申明

### DISCUSSION THREAD

**Discussion by cai-peng** [WeNeedHome/SummaryOfLoanSuspension#115] [Show and tell]
_"骆驼祥子的钱被抢了,黄包车被砸了,自己还挨了一顿打。还不允许他喊疼,祥子连呻吟一声都是错误的。“我们万千样子,要丢掉那些枷锁,让抢我们钱,砸我们车的人知道半子不在是那个任人宰割的羔羊_

**Comment by AntEffect** [comment]
![mmexport1657713646884.jpg](https://user-images.githubusercontent.com/12670263/178733642-aa67913e-4639-4c23-a73a-46536062725b.jpg)

**Reply by MarquessZhonggua** [reply]
说得好。

**Comment by answer-yida** [comment]
无产阶级失去的只是枷锁,而他们获得的将是整个世界 -------《共产党宣言》
